features.json

```diff
@@ -12,6 +12,7 @@
     "ember-container-inject-owner": true,
     "ember-htmlbars-local-lookup": null,
     "ember-application-engines": null,
-    "ember-glimmer": null
+    "ember-glimmer": null,
+    "ember-runtime-computed-uniq-by": true
```

Comment:
All new feature flags should have a value of 'null', we change to 'true' once the feature has been vetted and approved during one of our core team meetings.